src/mushroom-strategy.js

```diff
@@ -110,7 +106,7 @@ class MushroomStrategy {
             // Create a Title card for the current domain.
```

Comment:
If entities length is 1 but that entity is hidden via card options, the length is still 1 thus still creating a title card but with no entity cards.

Reply:
Yes, I'm aware of it.
The purpose of placing the PR was making you aware I'm working on something en to review the changes in the configuration.